zigbeetuya/integrationpluginzigbeetuya.cpp

```diff
@@ -69,6 +69,26 @@ bool IntegrationPluginZigbeeTuya::handleNode(ZigbeeNode *node, const QUuid &/*ne
 
         createThing(powerSocketThingClassId, node);
 
+        return true;
+    } else if (node->nodeDescriptor().manufacturerCode == 0x1141 && node->modelName() == "TS0201") {
```

Comment:
I see that you're checking the model name for == "TS0201". That's generally ok, but further down you're checking if the device has the "relative humidity cluster" or not to decide whether it's a H&T or just a Temp sensor. So my question is: Is it really the case that there are 2 different variants of the device with the same model name?